tests/wpt/mime-types.json

```diff
@@ -0,0 +1,397 @@
+[
+  "Basics",
```

Comment:
Can you also add the `tools/update-wpt.sh` file and run `chmod +x tools/update-wpt.sh` before you commit?